src/components/Navigation/Nav.tsx

```diff
@@ -97,21 +100,27 @@ export const Nav = ({ className }: NavProps) => {
       <nav className={classes.nav}>
         <ul className={classes.list}>
           <li className={classes.listItem}>
-            <Link to={ROUTE.home}>Inicio</Link>
+            <NavLink className={isActiveLink} to={ROUTE.home}>
+              Inicio
+            </NavLink>
```

Comment:
```suggestion
            <li>
            <NavLink className={({ isActive }) => classes.listItem(isActive)} to={ROUTE.home}>
              Inicio
            </NavLink>
          </li>
```

Y esto quedaría de esta forma.